cli/src/cli.rs

```diff
@@ -1108,17 +1116,40 @@ fn process_transfer(
     nonce_account: Option<&Pubkey>,
     nonce_authority: SignerIndex,
     fee_payer: SignerIndex,
+    derived_address_seed: Option<String>,
+    derived_address_program_id: Option<&Pubkey>,
 ) -> ProcessResult {
     let from = config.signers[from];
+    let mut from_pubkey = from.pubkey();
 
     let (recent_blockhash, fee_calculator) =
         blockhash_query.get_blockhash_and_fee_calculator(rpc_client, config.commitment)?;
 
     let nonce_authority = config.signers[nonce_authority];
     let fee_payer = config.signers[fee_payer];
 
+    let derived_parts = derived_address_seed.zip(derived_address_program_id);
```

Comment:
Hmm, you have `derived-address-program-id` Arg requires `derived-address-seed`, but not the other way around. So I was expecting `derived_address_program_id` to default to something (maybe the system program?) if only `derived_address_seed.is_some()`. Could be useful, although I'm not sure.
If your intent is both are required if either is provided, let's set that in the clap Args.

Reply:
Oh!  For some reason I was under the impression that setting one requirement linked them. I'll add the requirement